src/main/java/tasks/DeadLineTask.java

```diff
@@ -0,0 +1,127 @@
+package tasks;
+
+import java.time.LocalDate;
+import java.time.Month;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * The DeadlineTask Class inherits Task Class
+ * to support different specificities of a task
+ * as per input by user.
+ */
+public final class DeadLineTask extends Task {
+    /**
+     * Stores the type of task information
+     * that identifies a DeadlineTask.
+     */
+    private final String taskType = "[D]";
+
+    /**
+     * Stores the do-by-date information.
+     */
+    private final String dueDate;
+
+    /**
+     * Stores the do-by-date information as a LocalDate.
+     */
+    private LocalDate localDate;
```

Comment:
Should Javadoc of class members be specified on a single line?

Reply:
changed to single line, thanks for catching my mistake!